code/modules/hydroponics/grown/root.dm

```diff
@@ -21,7 +21,7 @@
 	desc = "It's good for the eyes!"
 	icon_state = "carrot"
 	bite_consumption_mod = 2
-	foodtypes = VEGETABLES
+	foodtypes = VEGETABLES | CARROT
```

Comment:
You need to add `CARROT` to carrot cake and carrot fries

There's also arguments to be made to add it to recipes that include carrots but that's up to you

Reply:
i thought i did, maybe I forgot to ctrl + s those files